modules/apps/cookies/cookies-banner-web/src/main/resources/META-INF/resources/js/CookiesUtil.js

```diff
@@ -46,5 +47,7 @@ export function setCookie(name, value) {
 }
 
 export function setUserConfigCookie() {
+	getOpener().Liferay.fire('cookieBannerSetCookie');
```

Comment:
I don't know a ton about cookie stuff, but would it make more sense to fire this on a granular level, firing inside of `setCookie`?

Also, I would suspect that a user would want to be able to get the value of the cookie when they are listening for this event. In this scenario, I don't think the user coudl get the value yet because technically the cookie isn't set yet and we aren't sending a value in the event.

So I would think either fire after setting the cookie OR we send the value in the event itself.

Reply:
This `setUserConfigCookie` function is called after the cookies are set in the browser, so we can make sure that it will work, and then we can get the cookies this way:

```
Liferay.on('cookieBannerSetCookie', () => {
      Liferay.Util.Cookie.get(Liferay.Util.Cookie.TYPES.PERFORMANCE) // true / false
})
```

I can create another function that will be called at the same time that `setUserConfigCookie` is called, I would have to add this new function to all the initial **banner buttons** and **banner configuration buttons**. What do you think? @bryceosterhaus

Reply:
Oh, got it! It makes total sense! I'll update the PR.

Reply:
Done @bryceosterhaus